zeebe/broker/src/main/java/io/camunda/zeebe/broker/partitioning/PartitionManagerImpl.java

```diff
@@ -544,4 +547,21 @@ public ActorFuture<Void> initiateScaleUp(final int desiredPartitionCount) {
 
     return result;
   }
+
+  @Override
+  public ActorFuture<Void> awaitRedistributionCompletion(
+      final int desiredPartitionCount,
+      final Set<Integer> redistributedPartitions,
+      final Duration timeout) {
+    final var result = concurrencyControl.<Void>createFuture();
+
+    brokerClient.sendRequestWithRetry(
```

Comment:
:thought_balloon: This retries very aggressively, every 10ms without any backoff. This would spam P1 with a lot of requests during redistribution and relocation. Instead of using the transport-level retry, you could also have the operation just fail. The dynamic-config module will then retry with exponential backoff. That will result in a lot of warning logs though which we could silence for certain operations, for example by marking the operation with an interface.

Reply:
Is the exponential backoff already built into when trying to apply an operation?

Reply:
Yes, see `ClusterConfigurationManagerImpl#logAndScheduleRetry`